package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="3">
   <name>mim_solvers</name>
-  <version>0.0.5</version>
+  <version>0.1.0</version>
```

Comment:
this should no appear before the v0.1.0 tag, and it will be added automatically by jrl-cmakemoduls release process